src/commands/createTransactionCastVotes.js

```diff
@@ -49,26 +49,38 @@ export const actionCreator = vorpal => async ({ options }) => {
 	const {
 		passphrase: passphraseSource,
 		'second-passphrase': secondPassphraseSource,
-		vote,
-		unvote,
+		votes,
+		unvotes,
 	} = options;
 
-	if (!vote && !unvote) {
+	if (!votes && !unvotes) {
 		throw new Error('At least one of votes and/or unvotes options must be provided.');
 	}
 
-	if (vote === unvote) {
+	if (votes === unvotes) {
 		throw new Error('Votes and unvotes sources must not be the same.');
 	}
 
-	const votes = vote ? await processVotesInput(vote.toString()) : null;
-	const unvotes = unvote ? await processVotesInput(unvote.toString()) : null;
+	const processedVotesInput = votes
+		? await processVotesInput(votes.toString())
```

Comment:
Something odd is happening here:
<img width="1296" alt="screen shot 2017-12-12 at 12 42 12" src="https://user-images.githubusercontent.com/11036220/33883009-64ee2c28-df3a-11e7-8316-eeaa6de79a34.png">

Reply:
This might be beyond the scope of this issue.